signalfx-org-metrics/README.md.jinja

```diff
@@ -11,43 +11,72 @@
 
 ## DESCRIPTION
 
-Splunk Infrastructure Monitoring generates a number of metrics you can use to monitor your usage of the application, including:
+Splunk Infrastructure Monitoring provides the following types of metrics that measure your usage:
 
--  Ingest-related metrics that characterize the volume and nature of data that you are sending to Infrastructure Monitoring, such as the number of datapoints that were sent or how many metric time series were created in response to the datapoints and dimensions sent.
+-  Ingest metrics that measure the data you're sending to Infrastructure Monitoring, such as the number of datapoints you've sent, or the number of metric time series (**MTS**) created from the datapoints and dimensions you sent
 
--  Usage or engagement metrics, such as the number of dashboards or charts in the organization.
+-  App usage or engagement metrics that measure your use of application features, such as the number of dashboards or charts in your organization
 
--  Metrics that tell you how the application pulls data on your behalf from cloud services like Amazon Web Services or Google Cloud Platform, such as the number of calls to each GCP Stackdriver client method or how many of the calls to the AWS CloudWatch API are being throttled by AWS.
+-  Cloud service integration metrics that measure how Infrastructure Monitoring pulls data from cloud services such as Amazon Web Services (**AWS**) or Google Cloud Platform (**GCP**), such as the number of calls to each GCP Stackdriver client method, or the number of the calls to the AWS CloudWatch API that AWS has throttled
 
-Your organization is not charged for these metrics, and they do not count against any limits.
+-  Metrics that measure your use of resources for which you can specify limits, such as the number of custom MTS you've created. These metrics
+   help you track usage and avoid going over your organization limits.
 
+You're not charged for these metrics, and they do not count against any limits.
 
-## USAGE
 
-Admins can see some of these values in built-in charts on the Organization Overview page. And, like any other metrics, you can display them on charts you build to monitor your system health.
+## USAGE
 
+Admins can view some of these metrics in built-in charts on the Organization Overview page. Like other metrics, you can display them in custom charts you create in order to monitor your system health.
 
-### About ByToken metrics
+### Metric pairs that provide total and ByToken values
 
 Some metrics send both a total value and a ByToken value, such as `sf.org.numAddDatapointCalls` and `sf.org.numAddDatapointCallsByToken`.
```

Comment:
```suggestion
Some metrics send both a total value and a `ByToken` value, such as `sf.org.numAddDatapointCalls` and `sf.org.numAddDatapointCallsByToken`.
```

Reply:
I think I should rewrite it as "### Metrics that provide values by token" and refer to the concept as by token rather than `ByToken`.

Reply:
Fixed.